scico/linop/radon_svmbir.py

```diff
@@ -56,13 +57,15 @@ def __init__(
             angles: Array of projection angles in radians, should be
                 increasing.
             num_channels: Number of pixels in the sinogram.
+            center_offset: Position of the image center relative to the center-of-rotation in units of (fractional) pixels
```

Comment:
Nitpick: ... relative to the center-of-rotation, in units ...

Reply:
Made the change